src/gui/mainwindow.cpp

```diff
@@ -141,6 +146,10 @@ MainWindow::MainWindow(QWidget *parent)
 {
     m_ui->setupUi(this);
 
+    Theme::Serialization::registerGuiColorProviders();
```

Comment:
Not in this PR, of course...
I didn't think about implementation, I have the idea itself. Now we have MainWindow as GUI manager, but this is wrong approach.